BMM.Core/Translation/en/main.json

```diff
@@ -330,6 +330,8 @@
     "OptionNotificationBadgeText": "Show a notification in home to help you remember to listen to the daily episode.",
     "OptionBibleStudyHeader": "Bible Study on Home",
     "OptionBibleStudyText": "Show your Bible Study status on the home screen",
+    "OptionRemoveBadgeOnStreakPointOnlyHeader": "Badge on Streak point",
```

Comment:
Can you change the text please:
Title: Strict badge requirements
Text: Only remove the Fra Kåre notification badge when you have earned the streakpoint

Reply:
Done